src/libraries/NetCoreAppLibrary.props

```diff
@@ -0,0 +1,160 @@
+<Project>
+  <PropertyGroup>
+    <NetCoreAppLibrary>
+      Microsoft.CSharp;
```

Comment:
Discussed this with @ericstj offline. We need a way to determine which library is part of the shared framework so that we can build the targeting pack first and later consume it in OOB libraries. There were at least two options:
a) add a target that returns the value which is called by src.proj/ref.proj/tests.proj. Big downside is cost and potential static graph restore issues.
b) Centralize the list of libraries that compose the shared framework (approach that I took). Downside: it's not defined per leaf anymore but in favor no additional build costs.

Reply:
I see... I think it helps with having a centralized list of what's inbox (it helps and sometimes people struggle with finding what's inbox and what's not). 

I guess the reason why we chose a Property instead of an Item (which is more readable), was for simplicity of figuring out if a project was inbox?

Reply:
> I see... I think it helps with having a centralized list of what's inbox (it helps and sometimes people struggle with finding what's inbox and what's not).

Jep, agree.

> I guess the reason why we chose a Property instead of an Item (which is more readable), was for simplicity of figuring out if a project was inbox?

I actually started with an Item but had to change to a property as other properties relied on IsNetCoreApp.

Reply:
I see, ok. Sounds good. Thanks for explaining the motivation of changing this 😄